### PR TITLE
Polish translation update 2020-08-12

### PR DESCRIPTION
**Pull request by mfilocha**
Synchronize Polish translation with upstream English changes up to fcd8af988cf3586e88f02fff9ddb776a8fc7047d.

### CODE REVIEW

File: content/pl/docs/concepts/overview/kubernetes-api.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 title: API Kubernetes
 content_type: concept
 weight: 30
+description: >
+  API Kubernetesa służy do odpytywania i zmiany stanu obiektów Kubernetesa.
+  Sercem warstwy sterowania Kubernetesa jest serwer API i udostępniane przez niego HTTP API. Komunikacja pomiędzy użytkownikami, różnymi częściami składowymi klastra oraz komponentami zewnętrznymi odbywa się poprzez serwer API.
```
Comment (kpucynski) [inline content/pl/docs/concepts/overview/kubernetes-api.md:7]:
```suggestion
  Sercem warstwy sterowania Kubernetesa jest serwer API i udostępniane przez niego HTTP API. Przez ten serwer odbywa się komunikacja pomiędzy użytkownikami, różnymi częściami składowymi klastra oraz komponentami zewnętrznymi.
```

File: content/pl/docs/concepts/overview/components.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 ---
-title: Składniki Kubernetes
+title: Składniki Kubernetesa
 content_type: concept
+description: >
+  Klaster Kubernetesa tworzą komponenty warstwy sterowania
```
Comment (nvtkaszpir) [inline content/pl/docs/concepts/overview/components.md:5]: Maybe `Klaster kubernetesa tworzą: komponenty ...`?

Comment (mfilocha) [reply content/pl/docs/concepts/overview/components.md]: Fixed.

**Pull request merged**